dist/cargo_test.sh

```diff
@@ -12,6 +12,16 @@ cargo_test_flags["rh-manifest-generator"]=""
 cargo_test_flags["prometheus-query"]=""
 cargo_test_flags["quay"]="--features test-net"
 
+declare -A cargo_nextest_flags
+cargo_nextest_flags["cincinnati"]=""
+cargo_nextest_flags["commons"]=""
+cargo_nextest_flags["graph-builder"]=""
+cargo_nextest_flags["policy-engine"]=""
+cargo_nextest_flags["metadata-helper"]=""
+cargo_nextest_flags["rh-manifest-generator"]="--no-tests=pass"
+cargo_nextest_flags["prometheus-query"]=""
+cargo_nextest_flags["quay"]=""
```

Comment:
```suggestion
declare -A cargo_nextest_flags
cargo_nextest_flags["rh-manifest-generator"]="--no-tests=pass"
```

works as well; however, I could not find in documentation the specified behavior when a key does not exist in an associative array.